test/utest/src/cuter_solver_tests.erl

```diff
@@ -114,8 +114,8 @@ create_logfile(Fname, As, CreateLogs) ->
 guard_true({_Dir, Fname, Python}) ->
   As = [p1],  % One argument (the type is irrelevant)
   Mapping = create_logfile(Fname, As, fun guard_true_logs/2),
-  {ok, [SolNormal]} = cuter_solver:solve({Python, Mapping, Fname, 42}),
-  {ok, [SolRev]} = cuter_solver:solve({Python, Mapping, Fname, 1}),
+  {ok, [SolNormal]} = cuter_solver:solve(cuter_solver:mk_solver_input(Python, Mapping, Fname, 42)),
```

Comment:
It's a bit of a detail, but do we have any policy about lines not being too long (e.g. more than 80 chars)?
if so, perhaps all these changes can be spread in two lines.

An alternative, since the code is very repetitive and identical in all these changed lines is to use a macro
```erlang
-define(MkInput(N), cuter_solver:mk_solver_input(Python, Mapping, Fname, N)).
```
or even
```erlang
-define(MkInput(N), cuter_solver:mk_solver_input(Python, Mapping, Fname, N)).
-define(Solve(N), cuter_solver:solve(?MkInput(N))).
```
and use it throughout.

Reply:
Another alternative, similar in spirit but cleaner, is to use the following macro:
```erlang
-define(Solve(Cmd, Ms, F, Id), cuter_solver:solve(cuter_solver:mk_solver_input(Cmd, Ms, F, Id))).
```

Reply:
Done.